Sources_v3/StreamChatUI/ChatChannel/ChatFileAttachmentListView+ItemView.swift

```diff
@@ -0,0 +1,179 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+extension ChatFileAttachmentListView {
+    open class ItemView: View, UIConfigProvider {
+        public var attachment: _ChatMessageAttachment<ExtraData>? {
+            didSet { updateContentIfNeeded() }
+        }
+
+        public var tapHandler: () -> Void = {}
```

Comment:
I have feeling, that all our views that have some kind of `onTap` / `tapHandler` etc, should be implemented as `Control` subclass.

Pros:
- it handles touch interactions for us
- highlighted/enabled/disabled/selected states out of the box
- kinda intended way to do it

Cons:
- target/action requires `objc` methods
- in general it feels weird to make something control for just tap handling

Thoughts?
cc @VojtaStavik